wordpress-mobile/WordPress-iOS
# Gutenberg: Update gutenberg to use RN version 0.59.0

This PR implements the changes made on https://github.com/wordpress-mobile/gutenberg-mobile/pull/741, meant to update React Native to 0.59.0

I made a few changes to Podfile to simplify future updates and to keep the react-native `podfile.json` dependencies always tagged, instead of pointing to `master` branch (or branches in general).

This will avoid problems where podfile keeps pointing to the same branch but the file itself was updated in that branch.

To test:

- Run `rake dependencies`
- Check that all react-native dependencies are updated.
- Checkout the `gutenberg-mobile` branch and run `metro` server.
- Run the WPiOS app and open a gutenberg instance.
- Check that everything works fine.

**Note:** Since the JS Bundle haven't been updated yet, this won't work without metro running.
